omod/src/main/resources/config.xml

```diff
@@ -54,6 +54,14 @@
 		<require_module version="${metadatamappingVersion}">
 			org.openmrs.module.metadatamapping
 		</require_module>
+
+		<require_module version="${reportingVersion}">
+			org.openmrs.module.reporting
+		</require_module>
+
+		<require_module version="${metadatadeployVersion}">
+			org.openmrs.module.metadatadeploy
+		</require_module>
```

Comment:
Hi Rafal, we need this entry in order to load the metadatadeploy module at the time of referenceApplication module is initializing. It is explained in this wiki page. [https://wiki.openmrs.org/display/docs/Requiring+another+module+in+your+module](https://wiki.openmrs.org/display/docs/Requiring+another+module+in+your+module)